src/Assets/Standard Assets/Bugsnag/Bugsnag.cs

```diff
@@ -285,6 +307,14 @@ public enum Severity {
         }
     }
 
+    public static void MapUnityLogToSeverity(LogSeverity unitySeverity, Severity bugsnagSeverity) {
+        if (customMapping.ContainsKey(unitySeverity)) {
+            customMapping[unitySeverity] = bugsnagSeverity;
+        } else {
+            customMapping.Add(unitySeverity, bugsnagSeverity);
+        }
```

Comment:
Ah, for some reason I thought this wasn't supported in .NET 2.0, but it is! Changed!